README.md

```diff
@@ -155,7 +161,10 @@ coffee_average_per_person = []
 
 ### Answer
 
-```rb
+
+p (people[0][:transactions][0][:amount]+people[0][:transactions][2][:amount])/2
+p (people[1][:transactions][2][:amount])/1
+p (people[2][:transactions][0][:amount]+people[2][:transactions][2][:amount])/2
```

Comment:
This should be dynamic, what happens if you can't see the list and don't know how many coffees this person purchased, or if this list had thousands of purchases in it? Will you try and count each one? Think of a way to make your program do all the heavy work